internal/host/host_test.go

```diff
@@ -996,33 +998,33 @@ var _ = Describe("UpdateInventory", func() {
 				expectedInstallationDisk: ""},
 			{testName: "No previous installation disk, one disk in inventory",
 				currentInstallationDisk:  "",
-				inventoryDisks:           []*models.Disk{{Name: diskName}},
-				expectedInstallationDisk: hostutil.GetDeviceFullName(diskName)},
+				inventoryDisks:           []*models.Disk{{ID: diskId, Name: diskName}},
```

Comment:
I think the `diskName` can be removed here, it's irrelevant for the test

Reply:
It adds more complexity to the test....

Reply:
why? just delete `Name: diskName` and I think it will work

Reply:
The path is built from the Name field so it makes it more complicated.
please see your next comment where you asked me to add the other_disk_id to the inventory while the test works without id so i aligned it